packages/@aws-cdk/core/test/bundling.test.ts

```diff
@@ -603,9 +603,12 @@ describe('bundling', () => {
 
   test('ensure correct Docker CLI arguments are returned', () => {
     // GIVEN
+    process.env.TEST_FIXTURE_ENV = 'abc';
     const fromSrc = DockerBuildSecret.fromSrc('path.json');
+    const fromEnvironment = DockerBuildSecret.fromEnvironment('TEST_FIXTURE_ENV');
 
     // THEN
     expect(fromSrc).toEqual('src=path.json');
+    expect(fromEnvironment).toEqual('env=TEST_FIXTURE_ENV');
```

Comment:
Could you please add a test case where the env variable isn't correctly set but `fromEnvironment` is called?